lib/trino-parquet/src/main/java/io/trino/parquet/reader/ParquetReader.java

```diff
@@ -132,8 +132,7 @@
     private final Optional<ParquetWriteValidation> writeValidation;
     private final Optional<WriteChecksumBuilder> writeChecksumBuilder;
     private final Optional<StatisticsValidation> rowGroupStatisticsValidation;
-    private final List<RowRanges> blockRowRanges;
```

Comment:
Why did this change to an array?

Reply:
We avoid the need for `listWithNulls`, using an array provides the same behaviour in a more straightforward way.